store/extensions/Outfit Copy/extension.json

```
@@ -0,0 +1,40 @@
+{
+    "title": "Outfit Copy",
+    "description": "Copy everyone's outfit.\nCopy someone \":copy x\"\nCombine outfits: \":outfitcombine x y\"\nCycle through the outfits of everyone in the room: \":outfitcycle\"\nTo stop: \":outfitstop\n",
+    "authors": [
+      {
+        "name": "altxnor",
+        "hotel": "",
+        "username": "",
+        "discord": "",
+        "twitter": ""
+      }
+    ],
+    "version": "0.1.0",
+    "categories": ["Fun", "Usability", "Others"],
+    
+    "source": "https://github.com/altxnor/outfit-copy",
+    "readme": "",
+    "releases": "",
+    
+    "stable": true,
+    
+    "framework": {
+      "name": "G-Node",
+      "version": "0.2.15"
+    },
+
+    "language": "Java",
+    "commands": {
+      "default": ["java", "-jar", "NodeExtensionSetup.jar", "-npm", "start", "-v", "15.0.0", "-c", "{cookie}", "-p", "{port}", "-f", "{filename}"]
+    },
+    
+    "compatibility": {
+        "systems": ["Linux", "Windows", "Mac"],
+        "clients": ["Nitro"]
```

I'm pretty sure this would also work on Flash (haven't tested it tho)
```suggestion
        "clients": ["Nitro", "Flash"]
```